Encoders/x265.py

```diff
@@ -62,4 +62,4 @@ def match_line(self, line: str):
         :param line: one line of text output from the encoder
         :return: match object from re.search matching the number of encoded frames"""
 
-        return re.search(r"^(\d+)", line)
+        return re.search(r"^\[.*\]\s(\d+)\/\d+", line)
```

Comment:
For reference, the old x265 line output looked like:
```
221 frames
```

And the new output (when frames are specified) looks like:
```
[6.0%] 60/1000 frames, 29.98 fps, 945.95 kb/s, eta 0:00:31
```